stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -539,49 +554,42 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
   /// accessing `T`.
   ///
   /// The following example allocates enough raw memory to hold four instances
-  /// of `Int8`, and then uses the `initializeMemory(as:at:count:to:)` method
+  /// of `Int8`, and then uses the `initializeMemory(as:repeating:count:)` method
   /// to initialize the allocated memory.
   ///
   ///     let count = 4
   ///     let bytesPointer = UnsafeMutableRawPointer.allocate(
-  ///             bytes: count * MemoryLayout<Int8>.stride,
-  ///             alignedTo: MemoryLayout<Int8>.alignment)
+  ///             byteCount: count * MemoryLayout<Int8>.stride,
+  ///             alignment: MemoryLayout<Int8>.alignment)
   ///     let int8Pointer = myBytes.initializeMemory(
-  ///             as: Int8.self, count: count, value: 0)
+  ///             as: Int8.self, repeating: 0, count: count)
   ///
   ///     // After using 'int8Pointer':
-  ///     int8Pointer.deallocate(count)
+  ///     int8Pointer.deallocate()
   ///
   /// After calling this method on a raw pointer `p`, the region starting at
-  /// `p + index * MemoryLayout<T>.stride` and continuing up to
-  /// `p + (index + count) * MemoryLayout<T>.stride` is bound to type `T` and
-  /// initialized. If `T` is a nontrivial type, you must eventually deinitialize
-  /// or move from the values in this region to avoid leaks.
+  /// `self` and continuing up to `p + count * MemoryLayout<T>.stride` is bound 
+  /// to type `T` and initialized. If `T` is a nontrivial type, you must 
+  /// eventually deinitialize or move from the values in this region to avoid leaks.
   ///
   /// - Parameters:
   ///   - type: The type to bind this memory to.
-  ///   - index: The offset from this pointer to the region of memory to be
-  ///     initialized with `value`, in the stride of type `T`. `index` must
-  ///     not be negative. The default is zero.
+  ///   - repeatedValue: The instance to copy into memory.
   ///   - count: The number of copies of `value` to copy into memory. `count`
-  ///     must not be negative. The default is `1`.
-  ///   - value: The instance to copy into memory.
-  /// - Returns: A typed pointer to the memory referenced by this raw pointer,
-  ///   offset by `index * MemoryLayout<T>.stride` bytes.
+  ///     must not be negative. 
+  /// - Returns: A typed pointer to the memory referenced by this raw pointer.
   @_inlineable
   @discardableResult
-  public func initializeMemory<T>(as type: T.Type, at index: Int = 0,
-    count: Int = 1, to value: T
+  public func initializeMemory<T>( 
+    as type: T.Type, repeating repeatedValue:T, count: Int
```

Comment:
Nit: space after `repeatedValue:`; remove space at the end of the previous line.